tests/unit/s2n_server_cert_verify_test.c

```diff
@@ -21,6 +21,8 @@
 #include "error/s2n_errno.h"
 #include "crypto/s2n_ecdsa.h"
 
+#include "tls/s2n_server_cert_verify.c"
```

Comment:
do you mean .h?

Reply:
the .c is required because `s2n_server_cert_read_and_verify_signature()` is not a public interface but is used in the test.